### PR TITLE
Refactor around TestPlan TreeView

### PR DESCRIPTION
**Pull request by atodorov**
cc @RMadjev

### CODE REVIEW

File: tcms/static/js/tcms_actions.js [hunk 1]
```diff
@@ -289,31 +289,32 @@ function updateCommentsCount(caseId, increase) {
   }
 }
 
-function previewPlan(parameters, action, callback) {
-  var dialog = getDialog();
+function previewPlan(planIds, action, callback) {
+  let realIds = Array();
+  planIds.split(',').forEach(function(element) {
+    element = element.trim();
+    if (element !== "") {
```
Comment (asankov) [inline tcms/static/js/tcms_actions.js]: this is anti-pattern in javascript

should be:
```javascript
if (element) {}
```

Comment (atodorov) [reply tcms/static/js/tcms_actions.js]: Any reference with docs? I think there are more patterns like this.

Comment (RMadjev) [reply tcms/static/js/tcms_actions.js]: I think its better not to use 'let' without first introducing some transcompiler(Babel). The var and let have different scope rules. Im not sure if in this case we really need 'let'.

Comment (asankov) [reply tcms/static/js/tcms_actions.js]: I will dug-up something later. 

Sure, there are. What we need is linters.

**Pull request merged**